File: k9mail/src/main/java/com/fsck/k9/pEp/ui/activities/SplashActivity.java
package com.fsck.k9.pEp.ui.activities;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import com.fsck.k9.activity.SettingsActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Intent intent = new Intent(this, SettingsActivity.class);
        startActivity(intent);
        finish();
    }
}

File: k9mail/src/main/java/com/fsck/k9/pEp/ui/activities/SplashActivity.kt
package com.fsck.k9.pEp.ui.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.fsck.k9.BuildConfig
import com.fsck.k9.activity.SettingsActivity
import com.fsck.k9.pEp.ui.activities.provisioning.ProvisioningActivity


class SplashActivity : AppCompatActivity(), SplashScreen {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, ProvisioningActivity::class.java))
        finish()
    }
}
